.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     tests:
         runs-on: ubuntu-latest
 
-        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, PDF: ${{ matrix.pdf }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, wkhtmltox: ${{ matrix.wkhtmltox }}"
```

Comment:
Why `wkhtmltox` instead of `whhtmltopdf`?

Reply:
I can't find how to install just a `whhtmltopdf`. It seems I have to install `wkhtmltox` that contains `wkhtmltoimage` also.

Reply:
After our internal discussion, I decided to rename it to `wkhtmltopdf` 👍️